packages/opentelemetry-exporter-stackdriver-trace/README.md

```diff
@@ -0,0 +1,98 @@
+# OpenTelemetry Stackdriver Trace Exporter
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+OpenTelemetry Stackdriver Trace Exporter allows the user to send collected traces to Stackdriver.
+
+[Stackdriver Trace](https://cloud.google.com/trace) is a distributed tracing system. It helps gather timing data needed to troubleshoot latency problems in microservice architectures. It manages both the collection and lookup of this data.
+
+## Setup
+
+Stackdriver Trace is a managed service provided by Google Cloud Platform.
+
+### Installation
+
+Install the npm package `@opentelemetry/exporter-stackdriver-trace`
+
+```shell
+$ npm install @opentelemetry/exporter-stackdriver-trace
+```
+
+### Authentication
+
+The Stackdriver Trace exporter supports authentication using service accounts. These can either be defined in a keyfile (usually called `service_account_key.json` or similar), or by the environment. If your application runs in a GCP environment, such as Compute Engine, you don't need to provide any application credentials. The client library will find the credentials by itself. For more information, go to <https://cloud.google.com/docs/authentication/>.
+
+#### Service account key
+
+If you are not running in a GCP environment, you will need to give the service account credentials to the exporter.
+
+```js
+const { StackdriverTraceExporter } = require('@opentelemetry/exporter-stackdriver-trace');
+
+const exporter = new StackdriverTraceExporter({
+  serviceName: 'your-application-name',
+
+  /** option 1. provide a service account key json */
+  keyFile: './service_account_key.json',
```

Comment:
I think `keyFile` and `keyFileName` are the same thing, do you think we should just keep one (`keyFile`)?

Reply:
they are the same. I kept the exact interface that is passed to google client libraries and rely the library to get the projectid and credentials, so I wanted to keep their exact parameters.